feature/mission/src/main/java/com/example/mission/graph/MissionViewModel.kt

```
@@ -0,0 +1,21 @@
+
+@HiltViewModel
+class MissionViewModel @Inject constructor(
+
+) : ViewModel() {
+    private val _eventChannel = Channel<MissionEvent>()
+    val eventChannel = _eventChannel.receiveAsFlow()
+
+
+    sealed class MissionEvent {
+
+    }
```

detekt (1.23.7)

[warning] The class or object MissionEvent is empty.

(detekt.empty-blocks.EmptyClassBlock)